templates/partials/info.html

```diff
@@ -1,14 +1,16 @@
 <aside id="yrstruly">
     <p>This is the {{ game }} script viewer, brought to you by:</p>
+
     <ul>
-        <li><a href="https://twitter.com/xkeepah">xkeeper</a>: Original creator</li>
+        <li><a href="https://twitter.com/xkeepah">xkeeper</a> (Original creator)</li>
         <li><a href="https://github.com/Grossley">Grossley</a></li>
         <li><a href="https://github.com/Jacky720">Jacky720</a></li>
         <li><a href="https://kocka.tech">KockaAdmiralac</a></li>
     </ul>
+
     <ul class="comma-list">
         {% for text, link in links.items() %}
-        <li><a href="{{ link }}">{{ text }}</a></li>
+            <li><a href="{{ link }}">{{ text }}</a></li>
```

Comment:
This changes the resulting HTML from:
```html
<ul>
    <li><a>...</a></li>
</ul>
```

to

```html
<ul>
        <li><a>...</a></li>
</ul>
```
which I don't think is right when we use four spaces for indentation elsewhere.